sungc/rasterio_funcs.py

```diff
@@ -72,20 +72,20 @@ def load_bands(band_list, scale_factor, apply_scaling):
     spectral_cube : numpy.ndarray
         multi-band array with dimensions of [nBands, nRows, nCols]
```

Comment:
If you're feeling up to it, you could update the variable names as well.  Not a hard requirement though, but it just makes it consistent.

Reply:
Latest commits have updated variable names in the comments